tests/test_bootalgo.py

```diff
@@ -201,10 +215,46 @@ def test_error_warnings():
   - know edge cases that lead to errors provide valuable info (e.g. WCR with 
     one param regressions)
   '''
+
+def test_data_is_list(data):
   
+  df, B = data
+  cluster = df['cluster'].values.tolist()
+  X = df[['intercept', 'X1', 'X2']].values.tolist()
+  Y = df['Y'].values.tolist()
+  R = np.array([0,1,0])
+
+  with pytest.raises(TypeError):
+    Wildboottest(X = X, Y = Y, cluster = cluster, bootcluster = cluster, R = R, B = B, seed = 12341)
+
+def test_seeds(data):
   
+  df, B = data
+  cluster = df['cluster']
+  X = df[['intercept', 'X1', 'X2']]
+  Y = df['Y']
+  R = np.array([0,1,0])
+  
+  results_dict = []
+
+  for s in range(1,10000):
+    for w in ts:
+      boot = Wildboottest(X = X, Y = Y, cluster = cluster, bootcluster = cluster, R = R, B = B, seed = s)
+      boot.get_scores(bootstrap_type = "11", impose_null = True)
+      boot.get_weights(weights_type = w)
+      boot.get_numer()
+      boot.get_denom()
+      boot.get_tboot()
+      boot.get_vcov()
+      boot.get_tstat()
+      boot.get_pvalue(pval_type = "two-tailed")
+      results_dict.append(boot.pvalue)
+        
+  results_series = pd.Series(results_dict)
+  mapd = results_series.mad() / results_series.mean()
+  assert  mapd <= .1 # make sure mean absolute percentage deviation is less than 10% (ad hoc)
```

Comment:
I think we could set a lower threshold here, something in the 1-3% range? Also, this test will likely run for a while, but it's definitely good to have it.

Reply:
We're actually getting that the percent change is 0.08. All the p-values are .12-.18, so inference is the same, but they're on average 8% apart. I cobbled together this metric as:

mean(|x-x.mean()|)/x.mean()

I'm not sure if it's a standard one, just something I made up. Am happy to change it to a different distance function and threshold though

Reply:
I'll take a closer look tomorrow / on Saturday - it's getting somewhat late for thorough review here in Germany =)